src/env/env.c

```diff
@@ -0,0 +1,94 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   env.c                                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: alopez-g <marvin@42.fr>                    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/10/15 18:07:51 by alopez-g          #+#    #+#             */
+/*   Updated: 2022/10/16 01:18:23 by alopez-g         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "env.h"
+
+//easy version
+void	env_unsetvar(char ***env, char *key)
+{
+	int		i;
+	int		esize;
+
+	i = 0;
+	esize = env_size(*env);
+	while (*(*env + i))
+	{
+		if (key_compare(*(*env + i), key))
+		{
+			free(*(*env + i));
+			*(*env + i) = *(*env + esize - 1);
+			*(*env + esize - 1) = NULL;
+			return ;
+		}
+		i++;
+	}
+}
+
+char	*env_getvar(char ***env, char *key)
+{
+	while (**env)
+	{
+		if (key_compare(**env, key))
```

Comment:
uselly with boolean func we use **is**_XXX()

Reply:
changed to **is_key,** I cant find a better name